src/box/lua/config/applier/box_cfg.lua

```diff
@@ -241,6 +241,8 @@ local function apply(config)
     box_cfg.cluster_name = names.group_name
     box_cfg.replicaset_name = names.replicaset_name
     box_cfg.instance_name = names.instance_name
+    box_cfg.instance_uuid = configdata._instance_uuid
+    box_cfg.replicaset_uuid = configdata._replicaset_uuid
```

Comment:
I prefer to use underscored fields from `configdata` only inside `configdata` itself. Can we add `configdata:uuids()` simarly to `configdata:names()` to even just add the UUIDs into the `:names()` result?

Reply:
Added to names().